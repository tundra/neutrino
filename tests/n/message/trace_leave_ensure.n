# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

def $leave_plain_ensure() => try null ensure leave.outer();
do $leave_plain_ensure();
