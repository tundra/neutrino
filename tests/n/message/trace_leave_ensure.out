--- backtrace ---
- leave.outer()
- ensure
- :leave_plain_ensure()
Error: %<condition: Signal(escape)>
